client/app/components/app-view/error-handler.js

```diff
@@ -1,27 +1,26 @@
+import { EventEmitter } from 'events';
+
 // eslint-disable-next-line import/prefer-default-export
-export class ErrorHandler {
+export class ErrorHandler extends EventEmitter {
```

Comment:
Why it needs to be an EventEmitter? What was wrong with just setting the `error` value when needed to show an error?

Reply:
Yep... :+1: